hypershift-operator/controllers/hostedcluster/hostedcluster_controller_test.go

```diff
@@ -1957,6 +1976,9 @@ func TestValidateReleaseImage(t *testing.T) {
 					},
 				},
 			},
+			releaseImageLoookup: func(_ context.Context, _ string, _ []byte) (*releaseinfo.ReleaseImage, error) {
```

Comment:
s/releaseImageLoookup/releaseImageLookup (typo?)

Reply:
It is, can I fix in my next commit (I plan to get rid of all fake replacing it with gomock generated)?